crates/holochain-conductor-runtime-ffi/src/runtime.rs

```diff
@@ -112,6 +112,31 @@ impl RuntimeFfi {
         })
     }
 
+    /// Setup app
+    pub async fn setup_app(
+        &self,
+        payload: InstallAppPayloadFfi,
+        enable_after_install: bool,
+    ) -> RuntimeResultFfi<AppAuthFfi> {
+        debug!("RuntimeFfi::setup_app");
+        let payload: InstallAppPayload = payload.try_into()?;
+
+        #[cfg(not(test))]
```

Comment:
This this is going to be needed in a lot of places then it might be worth hiding this behind a trait that can swap its behavior out, so you just have to have the test/not test logic in one place?

Reply:
Ah interesting. I will give that a try.

Reply:
Is this what you had in mind? https://github.com/holochain/android-service-runtime/pull/87/commits/8005d39fb6150aa2110abf2e46525fd7d15a11aa